CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+## Expressions
+- Added `hasValue` to check if a value is known in the context
```

Comment:
Are we sure this is the best name? How about `isDefined` or something instead? This would match the `null` vs `undefined`-distinction that JavaScript has, at least...

Reply:
Don't know, I asked in #apireviews and didn't get a response. Perhaps `isDefined` is better?!